linux-arm64/Dockerfile.in

```diff
@@ -1,46 +1,34 @@
 FROM dockcross/base:latest
-MAINTAINER Matt McCormick "matt.mccormick@kitware.com"
 
-
-# Add sources.list
-COPY sources.list /etc/apt/sources.list
-RUN dpkg --add-architecture arm64 ; apt-get update ; apt-get -y upgrade ; apt-get update ;
-RUN apt-get install -y \
-  libbz2-dev:arm64 \
```

Comment:
For now they are removed because `mips` and `s390x` containers don't install extra libraries. Makes sense IMO, because dockcross is supposed to provide the toolchain binaries, not user-specific dev libraries.

@thewtex, you agree? From reading the dockcross README, I would say that this is a typical usecase described in chapter `How to extend Dockcross images`?

Reply:
I wrote that chapter 😛 
Note that these were added by @thewtex , and AFAIK the goal of Crosstool is also to provide an image that will support most use cases out of the box. But it's not my project.

Reply:
Ah I didn't notice. Let's wait for @thewtex to shine light on why they were added initially. Would they still be needed now?

Reply:
These can be removed now. Previously the toolchain came from these Debian packages -- now it comes from Crosstool-ng.